mods/ts/languages/rules/en.ftl

```diff
@@ -344,52 +344,52 @@ actor-gadept =
    .description = Repairs or sells vehicles and aircraft.
 
 actor-garadr =
+   .name = GDI Radar
    .description = Provides an overview of the battlefield.
-    Can detect cloaked units.
+    Detects cloaked units.
```

Comment:
We had an inconsistent way of describing cloak detection in other changes in this PR, we should choose one for consistency.